scout/constants/variant_tags.py

```diff
@@ -49,6 +49,79 @@
 
 VARIANT_CALL = ("Pass", "Filtered", "Not Found", "Not Used")
 
+# Formatting for different filters in the vcf file
+VARIANT_FILTERS = {
```

Comment:
Cool, it's working nicely: (cancer demo case, all filters pass :)  )

![image](https://user-images.githubusercontent.com/28093618/123796465-70696d80-d8e5-11eb-8f0a-7e1290e7540a.png)